lib/aruba/cucumber/core.rb

```diff
@@ -1,4 +1,4 @@
-if Aruba::VERSION >= '1.0.0'
+if false && Aruba::VERSION >= '1.0.0'
```

Comment:
Personally I use both `rspec` and `cucumber` in projects. Sometimes I ran both and want the results to be availabe. That's the reason for this fix. Can you point me to the failure to see if there's something we can do about this?